project/Dependencies.scala

```diff
@@ -54,6 +56,12 @@ object Dependencies {
   val mongo4catsCirce = "io.github.kirill5k" %% "mongo4cats-circe" % V.mongo4cats
   val circe = "io.circe" %% "circe-core" % "0.14.10"
 
+  val otel4sCore =  "org.typelevel" %% "otel4s-core" % V.otel4s
+  val otel4sPrometheusExporter = "org.typelevel" %% "otel4s-sdk-exporter-prometheus" % V.otel4s
+  val otel4sSdk = "org.typelevel" %% "otel4s-sdk" % V.otel4s
+  val otelJavaRuntimeMetrics =  "io.opentelemetry.instrumentation" % "opentelemetry-runtime-telemetry-java17" % "2.5.0-alpha"
```

Comment:
You don't need this dependency anymore, I believe.

Reply:
yes, thanks!